uli-community/lib/uli_community/api/app_access_token.ex

```diff
@@ -0,0 +1,21 @@
+defmodule UliCommunity.Api.AppAccessToken do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "app_access_tokens" do
+    field :token_id, Ecto.UUID
+    field :token_name, :string
+    field :access_level, :string
```

Comment:
Lets make this terminology consistent with the oAuth language. Lets call it `scope`
Example of oAuth scopes for mastodon can be seen here for your review `https://docs.joinmastodon.org/api/oauth-scopes/#granular`

Instead of string, this can be an array of strings. Because a token might have multiple scopes.

In our current iteration we can start with just two scopes - `dataset` and `classify`. We don't have to create a UI to select scopes for now. Every token created will have both these scopes for now. 
Note : don't make these the default value at the ecto schema level. Set them by default at the controller level whenever a new token is created.